test/kickstart-templates/kickstart.ks.template

```diff
@@ -47,10 +47,6 @@ if [ -n "REPLACE_PUBLIC_IP" ]; then
     echo "    - REPLACE_PUBLIC_IP" >>/etc/microshift/config.yaml
 fi
 
-# Update the ostree server URL
-ostree remote delete edge
-ostree remote add --no-gpg-verify edge REPLACE_OSTREE_SERVER_URL
-
```

Comment:
I think the ostreesetup should point to /run/install so that the OS installation finishes. In post install stage, we can point it to "our" repos. 
Let's see if the image can be created this way...

Reply:
In the scenario tests, there is a web server with all of the commits and we install over the network, not from files in an ISO image. That's why this step is redundant.

Reply:
/lgtm